tests/framework/installer/ceph_installer.go

```diff
@@ -798,6 +802,49 @@ func (h *CephInstaller) removeClusterFinalizers(namespace, clusterName string) {
 	logger.Infof("removed finalizers from cluster %s", objectMeta.Name)
 }
 
+func (h *CephInstaller) removeMonConfigMapFinalizers(namespace, configMapName string) {
+	ctx := context.TODO()
+	configMap, err := h.k8shelper.Clientset.CoreV1().ConfigMaps(namespace).Get(ctx, configMapName, metav1.GetOptions{})
+	if err != nil {
+		logger.Errorf("failed to remove finalizer. failed to get monConfigMap. %+v", err)
```

Comment:
Please replace `monConfigMap` and `monSecret` with individual words throughout these helpers